src/index.js

```diff
@@ -1,34 +1,37 @@
-var React = require('react');
+import React from 'react';
+import PropTypes from 'prop-types';
 
 /**
  * This component is wrapped around any part of a page.
  * When that part of the page is clicked, it selects
  * the text within.
  */
-var ClickToSelect = React.createClass({
-  propTypes: {
-    children: React.PropTypes.any.isRequired
-  },
-  select: function(e) {
+class ClickToSelect extends React.PureComponent {
+  select = e => {
     e.preventDefault();
-    var range = document.createRange(),
-      sel = window.getSelection();
+    const range = document.createRange();
+    const sel = window.getSelection();
     range.selectNodeContents(this._target);
     sel.removeAllRanges();
     sel.addRange(range);
-  },
-  render: function() {
-    var self = this;
+  };
+
+  render() {
+    const self = this;
     return React.createElement(
```

Comment:
This feels ok to you, don't want to replace it with JSX now that we're processing? I'm fine either way.